docs/index.md

```diff
@@ -2,6 +2,10 @@
 
 Victory is an opinionated, but fully overridable, ecosystem of composable React components for building interactive data visualizations. The following tutorial should help you get started with Victory. For more advanced examples, check out [our guides].
 
+### Getting Started with Victory Native? 
+
+[Check out the native version of this getting started tutorial]
```

Comment:
@paulathevalley no, that one 404s for now, because it points to a route on the live site that will be added by this PR `/docs/native`